### PR TITLE
Fix qemu-img version; rest of code required

### PR DESCRIPTION
**Pull request by jayofdoom**
Taken from:
https://github.com/openstack/ironic-python-agent/commit/b0aaa530039f5e83e1d6a6354b6e99ddf8d5139e

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,9 +16,13 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends python2.7 python2.7-dev \
         python-pip qemu-utils parted hdparm util-linux genisoimage git gcc \
         bash coreutils tgt && \
+    apt-get --only-upgrade -t testing install -y qemu-utils
```
Comment (morgabra) [inline Dockerfile]: &&?

Comment (jayofdoom) [inline Dockerfile]: fixed via amended commit

**Comment by jimrollenhagen** [comment]
This isn't everything from that commit but feels like it might do what we expect. +1

**Comment by jimrollenhagen** [comment]
Much better, thanks. +1

**Pull request merged**